vowpalwabbit/example.h

```diff
@@ -36,6 +36,7 @@ const unsigned char dictionary_namespace = 135;    // this is \x87
 const unsigned char node_id_namespace = 136;       // this is \x88
 const unsigned char message_namespace = 137;       // this is \x89
 const unsigned char ccb_decision_namespace = 139;
+const unsigned char ccb_history_namespace = 140;
```

Comment:
A bit of an aside: Is it expected that the user never provide features in these namespaces? Or are we treating collisions as something that "might just happen" but rarely?

Reply:
All namespaces above 125 are reserved and it is assumed a user does not use them

Reply:
We may want to brainstorm how to deal with namespace names that start with a unicode character that is non-ASCII. Well out of the scope of this change, though.

Reply:
We will need to extend the range of values in that case. But yes that is a discussion that will need to happen eventually